index.js

```diff
@@ -32,12 +33,34 @@ module.exports = {
     'guard-for-in': 2,
     'indent': [2, 2, { 'SwitchCase': 1 }],
     'jsx-quotes': [2, 'prefer-single'],
-    'import/prefer-default-export': 1,
-    'import/named': 1,
-    'import/newline-after-import': 1,
-    'import/no-duplicates': 1,
-    'import/no-mutable-exports': 1,
-    'import/imports-first': 1,
+    'import/default': 0, // TODO: enable once module.exports is supported
+    'import/extensions': 2,
+    'import/first': 2,
+    'import/newline-after-import': 2,
+    'import/no-absolute-path': 2,
+    'import/no-anonymous-default-export': 1, // TODO: discuss if this should be a 2
```

Comment:
Similar to the previous comment, I think what this library does should be independent of `import-js`.